S21/Rongxiang-Zhang/lower.py

```diff
@@ -0,0 +1,60 @@
+import csv
+
+
+def wordlower(word,lan):
+
+    word=word
+
+    lan=lan
+
+    if(lan=='en'):
+        for letter in word:
+            letter=word.lower()
+        print(letter)
+
+
+    if(lan=='tr' or lan=='az'):
+
+        for letter in word:
+            if (letter=='i' or letter=='I'):
+                letter=word.replace('I',"ı").lower()
+                print(letter)
+            else:
+                print(letter.lower(),end='')
+
+
+    if(lan=='ga'):
```

Comment:
How do we account for ga-IE? Current this if statement will not count ga-IE into this logic. We could say if "ga" in lan, this make it take into account all "ga" that have the same logic